klasses/models.py

```diff
@@ -112,6 +113,21 @@ def total_due_by_next_deadline(self):
         return self.installment_set.filter(
             deadline__lte=next_installment.deadline).aggregate(price=models.Sum('amount'))['price']
 
+    def personal_price(self, user):
+        """
+        Returns the personal price (if any) or standard price for a klass
+
+        Args:
+            user(User): the user to get a price for
+
+        Returns:
+            Decimal: the price for the klass
+        """
+        personal_price = self.personal_prices.filter(user=user).first()
+        if personal_price:
```

Comment:
Can you change this to `personal_price is not None`?

Reply:
✅